### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by kukulich**
## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/components/hui-card-options.ts [hunk 1]
```diff
@@ -49,7 +49,9 @@ export class HuiCardOptions extends LitElement {
           </div>
           <div class="secondary-actions">
             <mwc-icon-button
-              title="Move card down"
+              title=${this.hass!.localize(
```
Comment (bramkragten) [inline src/panels/lovelace/components/hui-card-options.ts:52]:
```suggestion
              .title=${this.hass!.localize(
```

File: src/panels/lovelace/components/hui-card-options.ts [hunk 2]
```diff
@@ -59,7 +61,9 @@ export class HuiCardOptions extends LitElement {
               <ha-svg-icon path=${mdiArrowDown}></ha-svg-icon>
             </mwc-icon-button>
             <mwc-icon-button
-              title="Move card up"
+              title=${this.hass!.localize(
```
Comment (bramkragten) [inline src/panels/lovelace/components/hui-card-options.ts:64]:
```suggestion
              .title=${this.hass!.localize(
```

File: src/panels/lovelace/components/hui-card-options.ts (hunk 1 above)
Comment (bramkragten) [inline src/panels/lovelace/components/hui-card-options.ts:52]:
```suggestion
              .label=${this.hass!.localize(
```

File: src/panels/lovelace/components/hui-card-options.ts (hunk 2 above)
Comment (bramkragten) [inline src/panels/lovelace/components/hui-card-options.ts:64]:
```suggestion
              .label=${this.hass!.localize(
```

File: src/panels/lovelace/components/hui-entity-editor.ts [hunk 3]
```diff
@@ -47,14 +47,18 @@ export class HuiEntityEditor extends LitElement {
                 allow-custom-entity
               ></ha-entity-picker>
               <ha-icon-button
-                title="Move entity down"
+                title=${this.hass!.localize(
```
Comment (bramkragten) [inline src/panels/lovelace/components/hui-entity-editor.ts:50]:
```suggestion
                .label=${this.hass!.localize(
```

File: src/panels/lovelace/components/hui-entity-editor.ts [hunk 4]
```diff
@@ -47,14 +47,18 @@ export class HuiEntityEditor extends LitElement {
...
+                  "ui.panel.lovelace.editor.card.move_entity_down"
+                )}
                 icon="hass:arrow-down"
                 .index="${index}"
                 @click="${this._entityDown}"
                 ?disabled="${index === this.entities!.length - 1}"
               ></ha-icon-button>
               <ha-icon-button
-                title="Move entity up"
+                title=${this.hass!.localize(
```
Comment (bramkragten) [inline src/panels/lovelace/components/hui-entity-editor.ts:59]:
```suggestion
                .label=${this.hass!.localize(
```

File: src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts [hunk 5]
```diff
@@ -107,15 +107,19 @@ export class HuiStackCardEditor extends LitElement
                   </mwc-button>
                   <ha-icon-button
                     id="move-before"
-                    title="Move card before"
+                    title=${this.hass!.localize(
```
Comment (bramkragten) [inline src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts:110]:
```suggestion
                    .label=${this.hass!.localize(
```

File: src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts [hunk 6]
```diff
@@ -107,15 +107,19 @@ export class HuiStackCardEditor extends LitElement
...
+                    )}
                     icon="hass:arrow-left"
                     .disabled=${selected === 0}
                     @click=${this._handleMove}
                   ></ha-icon-button>
 
                   <ha-icon-button
                     id="move-after"
-                    title="Move card after"
+                    title=${this.hass!.localize(
```
Comment (bramkragten) [inline src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts:120]:
```suggestion
                    .label=${this.hass!.localize(
```

**Review by bramkragten — CHANGES_REQUESTED** [review]

**Comment by bramkragten** [comment]
So we need to make some choices, the `title` attribute is for the mouse over and I think will also be used by screen readers, the `label` property will set the `aria-label` attribute on the native button element inside the `mwc-button` element, but will not show as a mouseover. If we use both, I think the screen reader will read the text twice, as the title and the `aria-label` are not on the same element.

Could you maybe confirm this @steverep, @joshms123, @nickallan1? For example, the Lovelace Edit Menu button, is that read twice?

And for the Close Lovelace edit mode button, we have a different `title` and `aria-label` because we want to give more context for screen reader users, does that work well? 

And does a `title` or `aria-label` make any difference for a screen reader?

If we know that we have to decide what buttons we want to show a mouse over for and what buttons we only want to make available for screenreaders and for what buttons a screen reader user might need more info so we add both with different text. Every icon button should have at least a label or a title.

**Comment by nickallan1** [comment]
Not sure if I'm looking at the correct edit button. I go to the open lovelace menu, configure ui and I see an edit button for each of my cards. That edit button reads fine.
 
there's a few ways to get an accessible label. a lot of sites actually put a span inside the button element and position it off screen so its readable to a screen reader but not visible. take a look at  https://www.freecodecamp.org/news/how-to-make-your-fancy-svg-button-accessible-83c9172c3c15/
as one example.

but if you are asking specifically about choosing aria-label or title. In most cases go with aria-label.
@bramkragten

**Pull request closed without merging**